conformance: Add Airlock Microgateway report for v1.1.0 (#3408)
kubernetes-sigs/gateway-api: site-src/implementations.md

Modern application security is embedded in the development workflow and follows DevSecOps paradigms.
Airlock Microgateway protects your applications and microservices with the tried-and-tested Airlock security features against attacks, while also providing a high degree of scalability.

With [Airlock Microgateway 4.4][airlock-microgateway-v4.4], Airlock Microgateway introduces a sidecarless data plane mode
based on Gateway API to avoid the operational complexity of sidecars.

#### Features
- Kubernetes native integration with sidecar injection and Gateway API support
- Reverse proxy functionality with request routing rules, TLS termination and remote IP extraction
- Using native Envoy HTTP filters like Lua scripting, RBAC, ext_authz, JWT authentication
- Content security filters for protecting against known attacks (OWASP Top 10)
- API security features like JSON parsing, OpenAPI specification enforcement or GraphQL schema validation

#### Documentation and links
- [Product documentation][airlock-microgateway-documentation]
- [Gateway specific documentation][airlock-microgateway-guide]
- Check our [Airlock community forum][airlock-microgateway-community-support] for support

> - Extended support is available for paid licenses following [the premium support process][airlock-microgateway-premium-support]

Reviewer: It's great you are adding links about support, this is something very useful for users. We do have a request for you though: since Gateway API is an open-source project, we try to stay away when it comes to licenses and paid support. We'd like to ask you to rephrase these sentences by not mentioning "extended support", "paid license", or "premium support". We are fine with leaving the links here, but just be sure you refer to support in a generic way, without making distinctions between the community and EE versions.

> - Check our [Airlock community forum][airlock-microgateway-community-support] for support
> - Extended support is available for paid licenses following [the premium support process][airlock-microgateway-premium-support]

Reviewer: Maybe something along the lines of
```suggestion
- Check our [Airlock community forum][airlock-microgateway-community-support] and [support process][airlock-microgateway-premium-support] for support.
```
This way, all the links are preserved, without explicitly mentioning EE. Let us know what you think, if that's something you'd consider :)

Author: @mlavacca Thank you for your valuable feedback and your suggestion to rephrase our implementation description.
Looks good to me as well.
